Datamigratie.Common/Services/OpenZaak/Models/OzZaaktype.cs

```diff
@@ -1,14 +1,31 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using System.Runtime.Serialization;
 
 namespace Datamigratie.Common.Services.OpenZaak.Models
 {
     public class OzZaaktype
-    {
-        public required string Onderwerp { get; set; }
-        public required string Identificatie { get; set; }
+    {   
+        public required string Url { get; set; }
+
+        public string Id { get; private set; } = string.Empty;
+
+        [OnDeserialized]
+        public void OnDeserialized()
+        {
+            Id = ExtractUuidFromUrl(Url);
+        }
+
+        private static string ExtractUuidFromUrl(string url)
```

Comment:
Waar hebben we de uuid voor nodig? zouden we de url niet als identifier kunnen gebruiken? Als de uuid daadwerkelijk toegevoegde waarde heeft, laten we dan een Guid gebruiken ipv een string